extensions/markdown-language-features/src/features/preview.ts

```diff
@@ -462,6 +462,11 @@ export class DynamicMarkdownPreview extends Disposable {
 			baseRoots.push(vscode.Uri.file(path.dirname(base.fsPath)));
 		}
 
+		const workspaceRoots = vscode.workspace.workspaceFolders?.map(folder => folder.uri);
```

Comment:
Please merge this into the if conditional above since that already adds the current folder to the list

Reply:
No problem! 😄